Misc/NEWS.d/next/Library/2017-08-18-17-16-38.bpo-5001.gwnthq.rst

```diff
@@ -1,4 +1 @@
-Except for those in windows-exclusive sections (not changed due to lack of
-familiarity), all asserts in `multiprocessing` are now more informative, and
-some error types have been changed to more specific ones. - Allen W. Smith
-(drallensmith on github)
+Many asserts in `multiprocessing` are now more informative, and some error types have been changed to more specific ones.
```

Comment:
I thought I was supposed to sign it?

Reply:
No, you don't need to. You can add an entry in `Misc/ACKS`, though (keeping alphabetical order).

Reply:
Already done, but thank you.